### PR TITLE
ja translation updates and encoding fix

### PR DESCRIPTION
**Pull request by softark**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes(doc)
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | --
| Fixed issues  | --

### CODE REVIEW

File: docs/guide-ja/glossary.md [hunk 1]
```diff
@@ -1,73 +1,73 @@
...
+アプリケーション・オブジェクトはエントリ・スクリプトによってシングルトンとしてインスタンス化されます。
+アプリケーション・シングルトンはどの場所からでも `\Yii::$app` としてアクセス可能です。
 
-## assets �A�Z�b�g
+## assets アセット
 
-�A�Z�b�g�Ƃ̓��\�[�X�E�t�@�C����w��������̂ł��B
-�T�^�I�ɂ� JavaScript �܂��� CSS ��܂ރt�@�C���ł����AHTTP �ɂ��ăA�N�Z�X�\�Ȃ�̂Ȃ牽�ł�\���܂���Bt or CSS code but can be anything else that is accessed via HTTP.
+アセットとはリソース・ファイルを指し示すものです。
+典型的には JavaScript または CSS を含むファイルですが、HTTP によってアクセス可能なものなら何でも構いません。t or CSS code but can be anything else that is accessed via HTTP.
```
Comment (samdark) [inline docs/guide-ja/glossary.md]: Not translated?

Comment (softark) [reply docs/guide-ja/glossary.md]: Ohhhh, my bad. I forgot to erase the garbage.

**Comment by samdark** [comment]
Merged. Thanks!

File: docs/guide-ja/glossary.md (hunk 1 above)
Comment (softark) [reply docs/guide-ja/glossary.md]: I've just fixed it.

**Pull request merged**